price-alert-script/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:latest
+
+COPY requirements.txt . 
+
+RUN pip3 install -r requirements.txt
+
+COPY price-alert-script.py . 
+
```

Comment:
Could make a WORKDIR and copy everything into it using COPY . /[workdir name]. Not an issue, could just be more concise :)